### DISCUSSION TITLE
Getting a failure - yet our security is set more stringent than the test

### DISCUSSION THREAD

**Discussion by O365-Zende** [maester365/maester#543] [Ideas]
On a number of the tests I reviewed after failure, there was a theme of fail because it doesn't take into account we may have a setting set tighter than the recommendation.

I would suggest tests where item best practice = "x", maybe allow for item = "x" or less than or more than depending on the test.

This would allow companies that choose to be more demanding than the best practice to have those tests pass.

4 fail examples below
![Example Maester](https://github.com/user-attachments/assets/42f276a4-5975-41ac-8825-dce2418cb322)

**Comment by merill** [comment]
Tagging @Cloud-Architekt to review the EIDSCA tests
